css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
```

Comment:
is 16 too small?

Reply:
I updated the sizes to `28 / 32 / 36 / 42 / 48`, so the smallest font size for cases with initials doesn't get smaller than 12px. 

I also looked into what sized we use today and it's `32 / 36 / 48`. and we have `64 / 96` in profile, but I'm not feeling strongly about going that big.